packages/create/README.md

```diff
@@ -21,6 +21,12 @@ npx sap-ux
 ## add
 Calling `sap-ux add` allows adding a feature to a project.
 
+### html
+Calling `sap-ux add html` will add html files for local preview and testing to the project. It will use the configuration from the `preview-middleware` from the `ui5.yaml` as default but also allows pointing to a different yaml e.g.:
```

Comment:
Yes (https://github.com/SAP/open-ux-tools/blob/feat/1335/get-config/packages/create/src/cli/add/html.ts#L45).

Reply:
It was a proposal to change the JS-Doc...

Reply:
Please be more precise with what you suggest. It is still not clear ...

Reply:
Please change:

> It will use the configuration from the `preview-middleware` from the `ui5.yaml` as default 

to

> It will use the configuration from the ui5.yaml as default, as provided by the preview-middleware

Reply:
Fix comes with #2093